EasyPost.Tests/BetaFeaturesTests/ParametersTests/ParametersTest.cs

```diff
@@ -0,0 +1,152 @@
+using System.Collections.Generic;
+using EasyPost.BetaFeatures.Parameters;
+using EasyPost.Models.API;
+using EasyPost.Tests._Utilities.Annotations;
+using Xunit;
+
+namespace EasyPost.Tests.BetaFeaturesTests.ParametersTests
+{
+    public class ParametersTest
+    {
+        public ParametersTest()
+        {
+        }
+
+        #region Tests
+
+        [Fact]
+        [Testing.Function]
+        public void TestParametersToDictionary()
+        {
+            const string street = "388 Townsend St";
+
+            var parameters = new Addresses.Create
+            {
+                Street1 = street,
+                Street2 = "Apt 20",
+                City = "San Francisco",
+                State = "CA",
+                Zip = "94107",
+                Country = "US",
+            };
+
+            var dictionary = parameters.ToDictionary();
+
+            // Check that a dictionary was created correctly
+            Assert.NotNull(dictionary);
+
+            // Check that the dictionary contains "address" + default "verify" and "verify_strict" top-level keys
+            Assert.True(dictionary.ContainsKey("address"));
+            Assert.True(dictionary.ContainsKey("verify"));
+            Assert.True(dictionary.ContainsKey("verify_strict"));
+
+            // Check that the "address" sub-dictionary was created correctly
+            var addressData = dictionary["address"] as Dictionary<string, object>;
+            Assert.NotNull(addressData);
+
+            // Check that the "address" sub-dictionary contains the 6 elements we set + default "residential" boolean
+            Assert.True(addressData.ContainsKey("street1"));
+            Assert.True(addressData.ContainsKey("street2"));
+            Assert.True(addressData.ContainsKey("city"));
+            Assert.True(addressData.ContainsKey("state"));
+            Assert.True(addressData.ContainsKey("zip"));
+            Assert.True(addressData.ContainsKey("country"));
+            Assert.True(addressData.ContainsKey("residential"));
+
+            // Check that the "street1" key was set with the correct value
+            Assert.NotNull(addressData["street1"]);
+            Assert.Equal(street, addressData["street1"]);
+        }
+
+        [Fact]
+        [Testing.Parameters]
+        public void TestParametersToDictionaryWithSubDictionary()
```

Comment:
I'd really like us to be annotating our tests with docstrings to define what the expected outcome of the test is. These tests are pretty long, it would be good if there was a simple single human readable line saying "this test does this thing and we expect it to do xyz"

AKA: The name of the test doesn't necessarily match up with the contents of this test. I get two different stories by looking at each.